tests/dragonfly/proxy.py

```diff
@@ -60,7 +60,7 @@ async def start(self):
 
     def drop_connection(self):
         """
-        Randomally drop one connection
+        Randomly drop one connection
```

Comment:
in start server please use try except so we can see more clearly the errors

Reply:
I don't follow.. How would that make the errors more clear?

Reply:
print the exception

Reply:
f2f